.github/workflows/tests.yml

```diff
@@ -0,0 +1,32 @@
+name: "Tests"
```

Comment:
I think you can use "Continuous Integration Test", it's the same as us in New Order Service